scripts/_draw_info_display.py

```diff
@@ -0,0 +1,226 @@
+        mwv_x = classes.VehiclePhysics.moving_water_velocity().x
+        mwv_y = classes.VehiclePhysics.moving_water_velocity().y
+        mwv_z = classes.VehiclePhysics.moving_water_velocity().z
+        text += f"     MWV X: {round(mwv_x, digits)}\n"
+        text += f"     MWV Y: {round(mwv_y, digits)}\n"
+        text += f"     MWV Z: {round(mwv_z, digits)}\n\n"
+
+    if moving_water_velocity_xyz:
+        mwv_xyz = math.sqrt(classes.VehiclePhysics.moving_water_velocity().x ** 2
+                + classes.VehiclePhysics.moving_water_velocity().y ** 2
+                + classes.VehiclePhysics.moving_water_velocity().z ** 2)
+        text += f"   MWV XYZ: {round(mwv_xyz, digits)}\n\n"
+
+    if charges_and_boosts:
+        mt = classes.KartMove.mt_charge()
+        smt = classes.KartMove.smt_charge()
+        ssmt = classes.KartMove.ssmt_charge()
+        mt_boost = classes.KartMove.mt_boost_timer()
+        trick_boost = classes.KartMove.trick_timer()
+        shroom_boost = classes.KartMove.mushroom_timer()
+        if classes.KartParam.is_bike() == 0:
+            text += f"MT Charge: {mt} ({smt}) | SSMT Charge: {ssmt}\n"
+        else:
+            text += f"MT Charge: {mt} | SSMT Charge: {smt}\n"
+        text += f"MT: {mt_boost} | Trick: {trick_boost} | Mushroom: {shroom_boost}\n\n"
+
+    if checkpoints_and_completion:
+        lap_comp = classes.RaceInfoPlayer.lap_completion()
+        race_comp = classes.RaceInfoPlayer.race_completion()
+        cp = classes.RaceInfoPlayer.checkpoint_id()
+        kcp = classes.RaceInfoPlayer.max_kcp()
+        rp = classes.RaceInfoPlayer.respawn_point()
+        text += f" Lap%: {round(lap_comp, digits)}\n"
+        text += f"Race%: {round(race_comp, digits)}\n"
+        text += f"CP: {cp} | KCP: {kcp} | RP: {rp}\n\n"
+
+    if miscellaneous:
+        wheelie_frames = classes.KartMove.wheelie_frames()
+        wheelie_cd = classes.KartMove.wheelie_cooldown()
+        trick_cd = classes.KartJump.cooldown()
+        airtime = classes.KartMove.airtime()
+        oob_timer = classes.KartCollide.solid_oob_timer()
+        if classes.KartParam.is_bike() == 1:
+            text += f"Wheelie Length: {wheelie_frames}\n"
+            text += f"Wheelie CD: {wheelie_cd} | Trick CD: {trick_cd}\n"
+        else:
+            text += f"Trick CD: {trick_cd}\n"
+        text += f"Airtime: {airtime} | OOB: {oob_timer}\n\n"
+
+    if surface_properties:
+        is_offroad = classes.KartCollide.surface_properties().offroad > 0
+        is_trickable = classes.KartCollide.surface_properties().trickable > 0
+        kcl_speed_mod = classes.KartMove.kcl_speed_factor()
+        text += f"  Offroad: {is_offroad}\n"
+        text += f"Trickable: {is_trickable}\n"
+        text += f"KCL Speed Modifier: {round(kcl_speed_mod * 100, digits)}%\n\n"
+
+    if position:
+        pos_x = classes.VehiclePhysics.pos().x
+        pos_y = classes.VehiclePhysics.pos().y
+        pos_z = classes.VehiclePhysics.pos().z
```

Comment:
Reduce to 1 function call via `pos_x, pos_y, pos_z = classes.VehiclePhysics.pos()`